app/cdap/components/WranglerV2/RecipeStepsTable/styles.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import grey from '@material-ui/core/colors/grey';
+import { DataGrid } from '@material-ui/data-grid';
+import styled from 'styled-components';
+
+export const GridContainer = styled.div`
+  height: 188px;
+`;
+
+// the !important is used where higher specificity is not working due to pre-defined inline styles on the components
```

Comment:
This use of !important is far too liberal here - I haven't had to use important when dealing with material ui and if data-grid forces you to use important, then maybe you shouldn't be using data grid.

Why did we add data grid to this? Give me the concrete reasons because it looks like a basic grid that could've easily been built by our current libraries and our current components.

Reply:
I have removed the use of !important
I have removed the data-grid library from @material-ui and created a custom table using MUI components